tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -712,9 +710,8 @@ public function notAddsSecondContentTypeMetaTag(string $htmlBefore, string $html
     }
 
     /**
-     * @return string[][]
-     *
-     * @psalm-return array<string, array{0:string, 1:string}>
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint
+     * @array<string, array{0:string, 1:string}>
```

Comment:
Should be `@return array...` rather than `@array...`.  Perhaps then the `@phpcsSuppress` would not be needed.

Reply:
Ah, that's why PHPCS was complaining. Now everything makes sense. :-) Will change.

Reply:
Fixed.